package.json

```diff
@@ -2,19 +2,29 @@
   "private": true,
   "license": "MIT",
   "devDependencies": {
-    "@babel/core": "^7.2.0",
-    "@babel/preset-env": "^7.2.0",
-    "@babel/register": "^7.0.0",
+    "@babel/core": "^7.4.3",
+    "@babel/polyfill": "^7.4.3",
+    "@babel/preset-env": "^7.4.3",
+    "@babel/register": "^7.4.0",
+    "acorn": "^6.1.1",
+    "ansi-colors": "^3.2.4",
+    "babel-loader": "^8.0.5",
     "browser-sync": "^2.26.3",
     "chai": "^4.2.0",
     "chai-as-promised": "^7.1.1",
     "color": "^3.1.0",
+    "core-js": "^3.0.0",
     "deep-iterator": "^1.1.0",
     "deepmerge": "^2.2.1",
     "del": "^3.0.0",
+    "eslint": "^5.16.0",
+    "eslint-config-airbnb-base": "^13.1.0",
+    "eslint-plugin-import": "^2.16.0",
+    "fancy-log": "^1.3.3",
     "flat": "^4.1.0",
     "glob": "^7.1.3",
     "gulp": "^4.0.0",
+    "gulp-eslint": "^5.0.0",
```

Comment:
Now webpack is handling the transpilation as well as the bundling, should probably exchange for webpack's [eslint-loader](https://webpack.js.org/loaders/eslint-loader/) so that can handle the linting too.

Reply:
Will leave like this for now.